bom/application/pom.xml

```diff
@@ -5071,13 +5071,9 @@
             </dependency>
             <dependency>
                 <groupId>org.hibernate.search</groupId>
-                <artifactId>hibernate-search-mapper-orm-jakarta</artifactId>
+                <artifactId>hibernate-search-mapper-orm-orm6</artifactId>
```

Comment:
Hi @Sanne, when you'll be rebasing next time, could you please also change the `hibernate-search-mapper-orm-coordination-outbox-polling-jakarta` just above to be also an -orm6 version? Thanks!

Reply:
(done that)